pkg/process/metadata/parser/service.go

```diff
@@ -152,14 +152,30 @@ func (d *ServiceExtractor) extractServiceMetadata(process *procutil.Process) *se
 		}
 	}
 
-	exe := cmd[0]
 	// check if all args are packed into the first argument
 	if len(cmd) == 1 {
-		if idx := strings.IndexRune(exe, ' '); idx != -1 {
-			exe = exe[0:idx]
+		if idx := strings.IndexRune(cmd[0], ' '); idx != -1 {
```

Comment:
Don't think you need this `if` since `strings.Split` will return the string passed in if there are no matches for the separator.

Reply:
It was not really introduced part of this PR but I think you are right. I can change as suggested thanks